doc/configuring.md

```diff
@@ -0,0 +1,361 @@
+
+[toc]
+
+Note: using private genome collections currently requires [genome-grist#130](https://github.com/dib-lab/genome-grist/pull/130).
+
+## Overview
+
+genome-grist does the following:
+
+* downloads metagenome data from the SRA, if requested;
+* pre-processes and trims it;
+* runs `sourmash gather` on the metagenome, using one or more sourmash databases, to find a [minimum metagenome cover](https://www.biorxiv.org/content/10.1101/2022.01.11.475838v1);
+* retrieves the full genomes for any matches and executes a variety of mapping-based analyses;
+* incorporates taxonomy information for the genomes into the taxonomy summary report (if taxonomy reporting is requested).
+
+Much of the configuration for genome-grist is about where to find more information about matching genomes.
+
+For Genbank genomes, this is easy! But if you're providing your own genomes and taxonomy information, it's a bit trickier.
+
+## Using Genbank genomes
+
+For Genbank genomes, all the necessary information is available already, or automatically determined by genome-grist!
+
+sourmash already provides pre-built databases containing [all GTDB genomes (R06 rs202)](https://sourmash.readthedocs.io/en/latest/databases.html) as well as [all 700,000 Genbank microbial genomes from July 2020](https://github.com/sourmash-bio/sourmash/issues/1749#issuecomment-947920226).
+
+For genomes available through Genbank (aka with Genbank accessions), genome-grist does the genome retrieval automatically, so you don't need to have them downloaded already.
+
+Taxonomy spreadsheets are available for GTDB (at the databases page) and for Genbank 700k/July 2020 (link upon request).
+
+## Preparing information on private genomes
+
+If you want to use unpublished or private genomes with genome-grist, you'll need to provide your own sourmash database, your own set of genome files and information, and your own taxonomy spreadsheet. Luckily this is all pretty straightforward and we provide tools to help you! Read on!
+
+Note that you can absolutely combine Genbank with your own databases here, or just use your own databases. (If there are overlapping identifiers, the private genomes are chosen first; you might want to do this if you already have a bunch of the Genbank genomes downloaded already, for example.)
+
+### Choosing identifiers for your genomes
+
+You'll need to choose unique identifiers for your genomes. genome-grist requires that your identifier does not have a space, colon (:), or forward slash (/) in it; everything else should be fine.
+
+### Preparing your genome files
+
+You'll need one FASTA file per genome (gzip or bz2 compressed is fine). The filename doesn't matter. It's probably easiest if they're all in one directory, although this isn't necessary.
+
+We suggest naming at the first sequence in each FASTA file with the identifier at the start, space delimited - for example `MY_ID_1.1 first_sequence_name is very special` .
```

Comment:
Will redundant sequence IDs affect the results? The entire header is unique but redundant IDs.

Reply:
We don't enforce anything about the IDs in sourmash, and the way genome-grist is coded, overlaps/non-unique IDs will end up resolving to just one genome (I don't recall if it's the first encountered, or what - I'll take a look).

I'll think if there's a good place to put checks in for non-unique IDs. Should be.

Reply:
https://github.com/dib-lab/genome-grist/issues/137